tests/test_finetuning.py

```diff
@@ -8,11 +8,14 @@
 from lr_face.data import FaceImage, DummyFaceImage, FacePair, make_pairs, \
     FaceTriplet, make_triplets
 from lr_face.losses import TripletLoss
-from lr_face.models import TripletEmbeddingModel, EmbeddingModel
+from lr_face.models import TripletEmbeddingModel, EmbeddingModel, Architecture
+from lr_face.utils import fix_tensorflow_rtx
 from lr_face.versioning import Tag
 from tests.src.util import scratch_dir, get_tests_path
 
-SCRATCH_DIR = get_tests_path('scratch/finetuning')
+fix_tensorflow_rtx()
+
+SCRATCH_DIR = get_tests_path('scratch/finetuning/tmp')
```

Comment:
tmp in scratch lijkt me wat veel?

Reply:
Klopt ja, ik kreeg eerst een gekke foutmelding en dacht dat ik een map dieper in scratch moest om het op te lossen, maar dat was niet de oplossing :sweat: En daarna vergeten weg te halen. Is weer teruggezet!